.env

```diff
@@ -0,0 +1,101 @@
+# Docker Compose Variables
+COMPOSE_PROFILES=default
+
+# Host-Port Mappings, credentials for Atomfeed Sync across various services. Defaults to services running in docker.
+OPENMRS_HOST=openmrs
+OPENMRS_PORT=8080
+OPENMRS_ATOMFEED_USER=admin
+OPENMRS_ATOMFEED_PASSWORD=test
+OPENELIS_HOST=openelis
+OPENELIS_PORT=8052
+OPENELIS_ATOMFEED_USER=atomfeed
+OPENELIS_ATOMFEED_PASSWORD=AdminadMIN*
+
+# Shared Variables
+BAHMNI_CONFIG_PATH=
+
+#OpenMRS Environment Variables
+OPENMRS_IMAGE_TAG=latest
+OPENMRS_DB_IMAGE_TAG=demo-latest
+OPENMRS_DB_NAME=openmrs
+OPENMRS_DB_HOST=openmrsdb
+OPENMRS_DB_USERNAME=openmrs-user
+OPENMRS_DB_PASSWORD=password
+OPENMRS_DB_CREATE_TABLES='false'
+OPENMRS_DB_AUTO_UPDATE='true'
+OPENMRS_MODULE_WEB_ADMIN='false'
+OPENMRS_DEBUG='true'
+OPENMRS_UPLOAD_FILES_PATH=./openmrs-uploads
+BAHMNI_OPENMRS_MODULES_PATH=
+MYSQL_ROOT_PASSWORD=adminAdmin!123
+
+#Bahmni Web Environment Variables
+BAHMNI_WEB_IMAGE_TAG=latest
+BAHMNI_APPS_PATH=
+
+#Bahmni Proxy Environment Variables
+PROXY_IMAGE_TAG=latest
+
+#Postgres Environment Variables
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=welcome
+POSTGRES_IMAGE_TAG=latest
+POSTGRES_HOST=db
+POSTGRES_PORT=5432
+POSTGRES_URL=jdbc:postgresql://db:5432/health_information_user
+
+#Rabbitmq Environment Variables
+RABBITMQ_USERNAME=guest
+RABBITMQ_PASSWORD=guest
+RABBITMQ_IMAGE_TAG=3
+RABBITMQ_HOST=rabbitmq
+RABBITMQ_PORT=5672
+
+#Otp service Environment Variables
+OTP_SERVICE_IMAGE_TAG=latest
+OTP_SERVICE_URL=http://otp
+
+#hiu-ui Environment Variables
+HIU_UI_IMAGE_TAG=latest
+HIU_BACKEND_BASE_URL=http://hiu:8080
+HIU_UI_BASE_NAME=/
+
+#hip and hiu Environment Variables
+HIU_ID="Bahmni"
+HIU_NAME="Bahmni"
+HIU_IMAGE_TAG=latest
+HIU_CLIENT_ID=
+HIU_CLIENT_SECRET=
+
+HIP_URL=
+HIP_ID=
+HIP_IMAGE_TAG=latest
+HIP_SECRET=
+HIP_CLIENTID=
+
+GATEWAY_JWK_URL="https://dev.abdm.gov.in/gateway/v0.5/certs"
+GATEWAY_BASE_URL="https://dev.abdm.gov.in/gateway/v0.5"
+HIU_DB_INITIALIZER_IMAGE_TAG=latest
+CONSENT_MANAGEMENT_URL="https://dev.ndhm.gov.in/cm"
+CONSENT_MANAGEMENT_SUFFIX="@sbx"
+CONSENT_NOTIFICATION_URL="http://localhost:8003/consent/notification"
+DATA_PUSH_URL=
+IDENTITY_JWK_URL= https://dev.ndhm.gov.in/auth/realms/consent-manager/protocol/openid-connect/certs
+GATEWAY_URL="https://dev.abdm.gov.in/gateway"
+OPENMRS_URL="http://openmrs:8080/openmrs"
```

Comment:
Why doesn't this variable use the same variables that already exisit? OPENMRS_HOST, PORT, etc? (Line 5/6)

Reply:
All OPENMRS creds are on top of the .env file. Why is this one lower in the file?

Reply:
we have reused the same variable